config/credentials.yml.enc

```diff
@@ -0,0 +1 @@
+rkYJzMB7AupAhfAw5DOI92MeswDTVd9ZRp2xphSeLiuZ31AT4qE/6ay9XDF+w4eSK3nNNxUn0e0ivugSB+QLYFcOAoXuWz7EDDPyR9Lvp+CfBJ0sdQN1b/eZYEExki1ABXIb9i/SutMxqKrolTDlkWkgCIte09GSuwMP+eLiHWuaCyN1elmG/ifELOlahbCqKmNmNkzsPg1CaFXZS3nV5rCARoA/16nVSa3NIczS50FpP/IFNep3tCUXleB4qcp0egFH+FIEKk2d7hz1WP8rYKUMVj+XULAc1y+HtLBTr/K9JqFjBQAiUDCiQCxWlB6lDeZOaZeh1XKzSdviz4rwAp3712iIOey2fEs+z3nqhOrpLFA80c08ca8wzm6ThzF4kYxsZqnme+BtH3oLrpec+MTyxiEj9q60Rt4T--DoV/cDwsb4iIIiT8--MLl1lXmZ3TkyRo1Hod68hw==
```

Comment:
same as database.yml

Reply:
so do i need to create credenetials.yml.enc.example file and run git rm? 
or i jus add it into gitignore?